localplanning.services/.env.dev

```diff
@@ -1,4 +1,3 @@
 PUBLIC_PLANX_EDITOR_URL=http://localhost:3000
 PUBLIC_PLANX_GRAPHQL_API_URL=http://localhost:7100/v1/graphql
-PUBLIC_PLANX_REST_API_URL=http://localhost:7002
-PUBLIC_LPS_URL=http://localhost:4321
```

Comment:
This is no longer required as we're passing this value in via the `--site` flag.